api/v1beta1/types.go

```diff
@@ -275,13 +285,20 @@ type OscKeypair struct {
 }
 
 type OscVm struct {
-	Name                string   `json:"name,omitempty"`
-	ImageId             string   `json:"imageId,omitempty"`
-	KeypairName         string   `json:"keypairName,omitempty"`
-	PrivateIps          []string `json:"privateIps,omitempty"`
-	SecurityGroupsNames []string `json:"securityGroupsNames,omitempty"`
-	SubnetName          string   `json:"subnetName,omitempty"`
-	ResourceId          string   `json:"resourceId,omitempty"`
+	Name                string                    `json:"name,omitempty"`
+	ImageId             string                    `json:"imageId,omitempty"`
+	KeypairName         string                    `json:"keypairName,omitempty"`
+	VmType              string                    `json:"vmType,omitempty"`
+	VolumeName          string                    `json:"volumeName,omitempty"`
+	DeviceName          string                    `json:"deviceName,omitempty"`
+	SubnetName          string                    `json:"subnetName,omitempty"`
+	LoadBalancerName    string                    `json:"loadBalancerName,omitempty"`
+	PublicIpName        string                    `json:"publicIpName,omitempty"`
+	SubregionName       string                    `json:"subregionName,omitempty"`
+	PrivateIps          []OscPrivateIpElement     `json:"privateIps,omitempty"`
+	SecurityGroupsNames []OscSecurityGroupElement `json:"securityGroupsNames,omitempty"`
```

Comment:
Change to `SecurityGroupNames` (also in json)

Reply:
Ok